lib/init.coffee

```diff
@@ -2,11 +2,17 @@ module.exports =
   config:
     configurationPath:
       type: 'string'
-      title: 'the path to SwiftLint\'s configuration file that relative to project or absolute'
+      title: 'Configuration Path'
+      description: 'SwiftLint\'s configuration file that relative to project or absolute'
       default: '.swiftlint.yml'
+    additionalOptions:
+      type: 'string'
+      title: 'Additional Options'
```

Comment:
Same with this one. Atom generates a title based on the option name, capitalizing the first letter, and inserting a space whenever it finds a capital letter. So `additionalOptions` _should_ automatically be being translated into `Additional Options`.